e2e/config/vhd.go

```diff
@@ -264,3 +272,17 @@ func (id VHDResourceID) Short() string {
 	}
 	return str
 }
+
+func GetRandomAmd64VHD() *Image {
```

Comment:
nit: `GetLinuxAMD64VHD()`

Reply:
GetRandomLinuxAMD64VHD, I want to keep Random in the func name